### PR TITLE
add documentation for brickify

### PR DESCRIPTION
**Pull request by epaulson**
I wrote up some docs for the Brickify tool. I didn't have a working spell checker so you get to see how good I am at spelling, which will be especially fun because I'm on a crappy mac keyboard with a sticky ooo key.

I did NOT commit some changes that something made to docs/source - in particular, something changed docs/source/brickschema.rst but it wasn't me. I think that's autogenerated or created by 'make docs' and probably shouldn't be tracked? I can add it if necessary.

### CODE REVIEW

File: docs/brickify/table_handler.rst [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+However, as a special case, Brickify converts the following strings to booleans: 
+["TRUE", "true", "True", "on", "ON"] all become ``True``, and ["FALSE", "false", "False", "off", "OFF"] are converted to ``False``.
+
+An important note: the replacement text is not carried out on the substrings. At present, this will **not** work:
+
+:: 
+
+    conditions:
+      - |
+        {thresh} > 12 and '{has_reheat}'
```
Comment (shreyasnagare) [inline docs/brickify/table_handler.rst:112]: Just wanted to mention that the `conditions` block here is a list and you could add all conditions as items in the list as well. Something like (I think):
```yaml
    conditions:
       - {thresh} > 12 
       - '{has_reheat}'
```

That would be logically equivalent to having just one condition like `<cond1> and <cond2>`

**Review by shreyasnagare — APPROVED** [review]
Thanks @epaulson! Not gonna lie, this is a much better documentation than what I could've written. I've added some minor comments in the `suggestion` format so you can add them quickly. 

I think it would be useful to link some working examples (the ones present in the `tests/data/brickify` directory?).

File: docs/brickify.rst [hunk 2]
```diff
@@ -0,0 +1,134 @@
+Brickify tool
+=========
+
+The `brickify` tool is used to create Brick models from other data sources. 
+It is installed as part of the ``brickschema`` package.
+
+The `brickify` tool is built around the notion of **handlers** and **operations**.
+**Handlers** are pieces of code (written in Python) that the `brickify` tool uses to carry out **operations** that transform data.
+
+**Handlers** are how data is loaded by `brickify` and contain the code executes the translations that are specified by the **operations**.
```
[the remark this answers is not in the document]
Comment (epaulson) [reply docs/brickify.rst:10]: I'm going to go through these and resolve them in my local branch and push a new change, but I liked your suggestions!

**Comment by shreyasnagare** [comment]
@gtfierro Thoughts? I wish we had a [Netlify config for the docs](https://hangar.tech/posts/deploying-sphinx-to-netlify/) as well.

**Comment by gtfierro** [comment]
Netlify is neat, but I think ultimately unnecessary due to the automatic build tools that readthedocs already provides. I do want to try the configuration out --- @epaulson could you make another commit to this branch so i can try the autobuild?

**Comment by epaulson** [comment]
@gtfierro trivial change pushed!

**Comment by gtfierro** [comment]
Awesome! Looks like the RTD build happened automatically; can you give a peak through the generated docs (https://brickschema--66.org.readthedocs.build/en/66/brickify.html) to see if the formatting and indexing worked out the way you want?

**Comment by epaulson** [comment]
Yup, looks fine. Do the build logs from the RTD site show up anywhere? There are warnings from Sphinx in my local build, I think related to my hacking 'include' but I wasn't sure how else to have the nested sections while still having separate files.

**Comment by gtfierro** [comment]
Build log is [here](https://readthedocs.org/api/v2/build/14411474.txt). Not sure if there is a better way of getting to it than through my RTD account, but it does at least seem to be publicly visible

**Pull request merged**